Tests.Integration.Presentation.Web/GDPR/V2/DataProcessingRegistrationApiV2Test.cs

```diff
@@ -1531,9 +1531,9 @@ private CreateNewContractRequestDTO CreateContractWithDprUuids(Guid organization
             };
         }
 
-        private string CreateName()
+        private string CreateName(bool hasDanishCharacters = false)
         {
-            return $"{nameof(DataProcessingRegistrationApiV2Test)}{A<string>()}";
+            return $"{nameof(DataProcessingRegistrationApiV2Test)}{A<string>()}{(hasDanishCharacters ? "æøå" : "")}";
```

Comment:
I added the bool because CreateName is being used in order to create Users etc.

Reply:
users are allowed to have danish chars and should also work so no need for that condition